test/prototype_transforms_kernel_infos.py

```diff
@@ -573,12 +588,16 @@ def transform(bbox, affine_matrix_, format_):
         out_bbox = F.convert_format_bounding_box(
             out_bbox, old_format=datapoints.BoundingBoxFormat.XYXY, new_format=format_, inplace=True
         )
+
+        # new_spatial_size = F._geometry._compute_affine_output_size(FIXME, w=spatial_size_[1], h=spatial_size_[0])
+        new_spatial_size = spatial_size_
```

Comment:
I think it is OK to pass output size as parameter instead recomputing it

Reply:
Not sure I understand. Are you saying we don't need to compute here?

Reply:
Yes, you can remove commented code and keep `spatial_size` as arg of the function